python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -975,6 +1023,13 @@ def convert_div_scalar(node, **kwargs):
     """
     return scalar_op_helper(node, 'Div', **kwargs)
 
+@mx_op.register("_rdiv_scalar")
+def convert_rdiv_scalar(node, **kwargs):
+    """Map MXNet's _rdiv_scalar operator attributes to onnx's Div operator.
+    Creates a new node for the input scalar value, adds it to the initializer
+    and return multiple created nodes.
+    """
+    return scalar_op_helper(node, 'Div', **kwargs)
 
```

Comment:
could you add _power_scalar too and the relevant test case.
once this is added tests with *bcast* can be enabled